acinclude/check_sst_core.m4

```diff
@@ -56,6 +56,7 @@ if test "X$have_integrated_core" = "Xyes"; then
   SUMI_CPPFLAGS="$SST_INCLUDES"
   AC_SUBST(SST_CPPFLAGS)
   CPPFLAGS="$SAVE_CPPFLAGS"
+  SST_CXXFLAGS=`echo "$SST_CXXFLAGS" | sed s/-std=c++11//g | sed s/-std=c++14//g | sed s/-std=c++1y//g | sed s/-std=c++1z//g`
```

Comment:
Might I suggest: sed "s/\(-std=c++1[1,4,y,z]\)//g"

Reply:
I would much prefer to use sed regular expressions to do this, but they are not universally stable across versions.  The above does not work on Mac.